src/controllers/issuer.ts

```diff
@@ -255,8 +257,8 @@ export class IssuerController {
 				if (serviceEndpoint) {
 					didDocument.service = [
 						{
-							id: `${didDocument.id}#service-1`,
-							type: 'service-1',
+							id: didDocument.id+"#"+serviceId,
```

Comment:
Let's do it in formatting style, like:
`${didDocuemnt.id}#${serviceId}`

Reply:
Done! ✅